src/mapblock_mesh.cpp

```diff
@@ -559,33 +547,34 @@ static void makeFastFace(TileSpec tile, u16 li0, u16 li1, u16 li2, u16 li3,
 		break;
 	}
 
-	for(u16 i=0; i<4; i++)
-	{
+	for (u16 i = 0; i < 4; i++) {
 		vertex_pos[i] = v3f(
-				BS/2*vertex_dirs[i].X,
-				BS/2*vertex_dirs[i].Y,
-				BS/2*vertex_dirs[i].Z
+				BS / 2 * vertex_dirs[i].X,
+				BS / 2 * vertex_dirs[i].Y,
+				BS / 2 * vertex_dirs[i].Z
 		);
 	}
 
-	for(u16 i=0; i<4; i++)
-	{
+	for (u16 i = 0; i < 4; i++) {
 		vertex_pos[i].X *= scale.X;
 		vertex_pos[i].Y *= scale.Y;
 		vertex_pos[i].Z *= scale.Z;
 		vertex_pos[i] += pos;
 	}
 
 	f32 abs_scale = 1.0;
-	if     (scale.X < 0.999 || scale.X > 1.001) abs_scale = scale.X;
-	else if(scale.Y < 0.999 || scale.Y > 1.001) abs_scale = scale.Y;
-	else if(scale.Z < 0.999 || scale.Z > 1.001) abs_scale = scale.Z;
+	if     (scale.X < 0.999 || scale.X > 1.001)
```

Comment:
the former is an acceptable exception to the code style IMO (if you add a space after `else if`)